distribution/sender/routes/database-event-reader.xml

```diff
@@ -0,0 +1,115 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+
+    <route id="db-event-reader">
+        <from uri="scheduler:retry?initialDelay=1000&amp;delay=10000" />
+  
+		<log message="Fetching events in the staging table" loggingLevel="DEBUG" />
+    
+        <toD uri="jpa:DebeziumEvent?query=SELECT dE.id FROM DebeziumEvent dE ORDER BY event_time_stamp ASC" />
```

Comment:
Order by dateCreated

Reply:
I thought "event_timestamp" was more accurate. Anyway, I'll change it